src/main/java/duke/exception/EmptyCommandException.java

```diff
@@ -0,0 +1,5 @@
+package duke.exception;
+
+public class EmptyCommandException extends DukeException{
```

Comment:
```suggestion
public class EmptyCommandException extends DukeException {
```
Similar for the other `DukeException` child classes.